app-android/src/main/java/io/github/droidkaigi/confsched2023/KaigiApp.kt

```diff
@@ -111,6 +111,9 @@ private fun KaigiNavHost(
             onTimetableItemClick = navController::navigateToTimetableItemDetailScreen,
             onNavigateToBookmarkScreenRequested = navController::navigateToBookmarkScreen,
             onLinkClick = externalNavController::navigate,
+            onRoomClick = {
+                navController.popBackStack(navController.graph.startDestinationId, false)
```

Comment:
Well, it's difficult to say but what if we create `navController::navigateToMainFloorMapScreen`?
I think we can see parent navController to access getBackStackEntry 👀

Reply:
Once I was able to return to the start destination with the following code.

```kotlin
fun NavController.navigateToMainFloorMapScreen() {
    getBackStackEntry(timetableItemDetailScreenRoute).destination.parent?.startDestinationId?.let {
        navigate(it)
        // TODO navigateMainFloorMap
    }
}
```

However, I have not figured out how to move to FloorMapScreen from here without having to make any major changes.
Do you already have any ideas?

Reply:
I'm not sure about this. But I think we can pass the `route` parameter to NavHost and we may be able to pass /main/floorMap.

Reply:
@takahirom 
I have tried the suggested method, but unfortunately the app crashes. 🙇 
We still think that as long as NavHost is split into two, it is unfortunately difficult to achieve this functionality other than this current implementation. 🙇